pkg/apis/core/v1alpha1/build_types.go

```diff
@@ -19,6 +19,7 @@ type BuildBuilderSpec struct {
 	// +patchStrategy=merge
 	// +listType
 	ImagePullSecrets []corev1.LocalObjectReference `json:"imagePullSecrets,omitempty" patchStrategy:"merge" patchMergeKey:"name" protobuf:"bytes,15,rep,name=imagePullSecrets"`
+	Kind             string                        `"json:kind,omitempty"`
```

Comment:
This might not be needed for the build resource since the `name` `image` and `kind` are passed in to builds via flags?

Reply:
maybe `Name` needs to be here too